tests/app/parser/test_metadata_parser.py

```diff
@@ -144,6 +144,28 @@ def test_deserialisation_iso_8601_dates(fake_metadata_runner):
     assert isinstance(claims["birthday"], str)
 
 
+def test_deserialisation_iso_8601_datetime(fake_metadata_runner):
+    claims = validate_runner_claims(fake_metadata_runner)
+
+    assert claims["response_expires_at"] == "2021-11-22T15:34:54+00:00"
```

Comment:
I think we should also test with Zulu time i.e `2021-11-22T15:34:54Z` since that is a valid iso8601 format and Marshmallow support it.

Reply:
It does work with `Z` but I specifically avoided testing all possible incidentally supported formats because Mark agreed on the single specific format it should be, which is `2021-11-22T15:34:54+00:00`

Reply:
I get that but given the schema definition doc that got merged says `An ISO_8601 formatted date-time` I think Zulu should be tested otherwise even if it works we won't have confidence in it. They may not use Zulu but if the spec is 8601 then I think it should be tested given there is no caveat made in the doc about not using zulu. I don't think there is anything else to test, not sure what you mean by `all possible incidentally supported formats`?. Zulu is distinctly different to the offset, so just 2 tests is all I see as needed.

Reply:
Mark appears to have gone to pains to pin down the exact single format we would support: https://surveydatacollection.slack.com/archives/CLR17KQKU/p1636726964118200?thread_ts=1636726041.118000&cid=CLR17KQKU

Reply:
Had a discussion with @pricem14pc about this and came to the decision for us to add a test for Zulu instead of updating the document specification to says we don't support Zulu.